components/Db.js

```diff
@@ -84,80 +84,85 @@ const foundItems = [
     name: "Camera",
     description: "Camera underwater",
     img: "/assets/foundItems/one.jpeg",
-    id: 1,
+    id: "14",
   },
   {
     name: "Backpack",
     description: "Green and black backpack found in the woods",
     img: "/assets/foundItems/two.jpeg",
-    id: 2,
+    id: "15",
   },
   {
     name: "Diamonds",
     description: "We found a ton of diamonds!!!",
     img: "/assets/foundItems/three.jpeg",
-    id: 3,
+    id: "16",
   },
   {
     name: "One shoe",
     description: "Found one shoe. Where might the other one be?",
     img: "/assets/foundItems/four.jpeg",
-    id: 4,
+    id: "17",
   },
   {
     name: "Ring",
     description: "Found the most beautiful ring with blue stone attached",
     img: "/assets/foundItems/five.jpeg",
-    id: 5,
+    id: "18",
   },
   {
     name: "Key",
     description: "Ancient Key found in the Woods",
     img: "/assets/foundItems/six.jpeg",
-    id: 6,
+    id: "19",
   },
   {
     name: "Keyboard",
     description: "Oldschool Keyboard with Keys missing",
     img: "/assets/foundItems/seven.jpeg",
-    id: 7,
+    id: "20",
   },
   {
     name: "Shoe",
     description: "Found one shoe in an abandoned building",
     img: "/assets/foundItems/eight.jpeg",
-    id: 8,
+    id: "21",
   },
   {
     name: "Handshoe",
     description: "One handshoe found on a parking bench",
     img: "/assets/foundItems/nine.jpeg",
-    id: 9,
+    id: "22",
   },
   {
     name: "Camera",
     description: "Black camera found in the woods",
     img: "/assets/foundItems/ten.jpeg",
-    id: 10,
+    id: "23",
   },
   {
     name: "Football",
     description: "Found a football",
     img: "/assets/foundItems/eleven.jpeg",
-    id: 11,
+    id: "24",
   },
   {
     name: "Golden coins",
     description: "Found some golden coins. Wont give em back.",
     img: "/assets/foundItems/twelve.jpeg",
-    id: 12,
+    id: "25",
   },
   {
     name: "Golden Jewelry",
     description:
       "Found jewelry and intend to keep it. Just posting here for recognition.",
     img: "/assets/foundItems/thirteen.jpeg",
-    id: 13,
+    id: "26",
```

Comment:
if you don't want to edit every id manually you could just install nanoid for random unique ids.. super easy to use (did it myself and can recommend it) 
https://github.com/ai/nanoid

Reply:
For the future i will definitely rethink this. Currently the dummy data remains and will not be greatly expanded for the time being. Thanks for the good idea!